tests/resources_t/test_poolfactory.py

```diff
@@ -126,26 +126,24 @@ def test_create_composite(
     @patch("tardis.resources.poolfactory.BatchSystemAgent")
     @patch("tardis.resources.poolfactory.SiteAgent")
     def test_create_drone(self, mock_site_agent, mock_batch_system_agent, mock_drone):
-        self.assertEqual(
```

Comment:
This change is not directly related to the reason for the pull request. Instead it fixes a broken unittest. Actually, the `test_create_drone` test did not test anything.